src/api/urls.py

```diff
@@ -12,4 +14,6 @@
     path("me/", MeView.as_view(), name="me"),
     path("rides/<int:id>/", RideView.as_view(), name="ride"),
     path("rides/", RidesView.as_view(), name="rides"),
+    path("requests/<int:id>/", RequestView.as_view(), name="request"),
```

Comment:
take a look at how https://docs.djangoproject.com/en/4.1/intro/tutorial01/ does URLs, might as well get one of our Django pods to make the `api/urls.py` file more lightweight the right way